tests/ml_commons/test_ml_commons_client.py

```diff
@@ -237,6 +237,64 @@ def test_DEPRECATED_integration_pretrained_model_upload_unload_delete():
             raised = True
         assert raised == False, "Raised Exception in deleting pretrained model"
 
+def test_predict():
+    input_json = {
```

Comment:
Can you consider taking `iris_index` as an argument here. Since, that is a fixture, you would have the iris data in the opensearch in an index. `iris_index` variable will have the actual name of the index here.. So, `input_index` can be `iris_index`